lib/hexagon/extensions/plug/local_store.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Hexagon.Extensions.Plug.LocalStore do
+  @moduledoc """
+  An easy local store.
+
+  It serves files at `http://localhost:4002` and stores
+  them at `priv/data/packages` (both configurable). This is
+  an easy implementation for development or a simple
+  production deployment.
+  """
+
+  use Plug.Router
+
+  @enabled? Application.compile_env(:hexagon, :package_store)[:module] ==
+              Hexagon.Packages.Store.Local
```

Comment:
Interesting 🤔

Reply:
Yeah, I need to switch these to not compile env so they can be set at runtime.

Reply:
First time I've seen this pattern

Reply:
I think I'll probably keep this as is for the first iteration. But because the put and delete methods are unprotected, it will be reworked in the future so it's "production grade"